### ISSUE TITLE
RHVoice on Android: RHVoice cannot be used with the Jieshuo screen reader

### ISSUE THREAD

**Issue by k-kolev1985** [RHVoice/RHVoice#630]
### Description
There is a chinese screen reader for Android, called "Jieshuo". Its GitHub page is here: https://github.com/nirenr/jieshuo, though it seams to be in chinese.

In its free version Jieshuo uses the default system TTS engine, the way TalkBack does.

When I set "RHVoice" as the system's default TTS engine, Jieshuo stays silent until I restart it. After I restart it, even though "RHVoice" is selected, the TTS engine that speaks is "Speech Services by Google". I have at least one russian voice downloaded for "RHVoice" - "Artemiy", but still the problem occurs. The TTS engine's language in the system is set to "Use system language", but even if I set it to "Russian", it still does not work. In the system's TTS settings, with "RHVoice" selected and the TTS language set to "Russian", I can successfully play a sample.

I have 7 TTS engines installed in total on my phone (incl. "RHVoice"), but none of the others exhibit such a behavior - only "RHVoice".

### Test environment:
- Device: Poco X4 Pro 5G, running Android 11 with MIUI Global 13.0.9 for Poco with Android security patch from 2022-08-01.
- Locale: Display language is set to "Bulgarian" and region is set to "Bulgaria".
- Jieshuo version: 20220901 international.
- RHVoice version: 1.8.2.

**Comment by zstanecic** [comment]
Hi Kostadin,

This is not a bug of RHVoice, but it is a known bug of Jieshuo. To use any voice with Jieshuo, you need to set the phone language to the language you want to use in RHVoice.

So, for example, if you want to use RHVoice in Russian, you need to set the phone in Russian.

Best,

Zvonimir

**Comment by k-kolev1985** [comment]
Hi, Zvonimir,

I understand. Thanks for the clarification.

And now that I think about it, it makes sense - all the other 6 TTS engines that I have installed on my device do support bulgarian and therefore work with Jieshuo. The only exception is RHVoice, which does not support bulgarian and does not work with Jieshuo. And my phone's display language as I said is set to bulgarian.
